### ISSUE TITLE
[terra-tabs] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-framework#1174]
# Bug Report

The following themeable variables are missing:

**clinical-lowlight-theme**

--terra-tabs-collapsed-modular-tab-bar-border-bottom-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 26, column: 6)
--terra-tabs-collapsed-modular-tab-bar-border-bottom-width - packages/terra-tabs/lib/ModularTabs.module.scss (line: 28, column: 6)
--terra-tabs-modular-active-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 72, column: 8)
--terra-tabs-modular-active-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 73, column: 8)
--terra-tabs-modular-active-focus-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 105, column: 6)
--terra-tabs-modular-active-focus-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 106, column: 6)
--terra-tabs-modular-active-focus-border-radius - packages/terra-tabs/lib/ModularTabs.module.scss (line: 107, column: 6)
--terra-tabs-modular-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 23, column: 4)
--terra-tabs-modular-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 65, column: 6)
--terra-tabs-modular-background-position - packages/terra-tabs/lib/ModularTabs.module.scss (line: 66, column: 6)
--terra-tabs-modular-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 68, column: 6)
--terra-tabs-modular-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 69, column: 6)
--terra-tabs-modular-disabled-hover-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 85, column: 8)
--terra-tabs-modular-extended-flex - packages/terra-tabs/lib/ModularTabs.module.scss (line: 59, column: 8)
--terra-tabs-modular-focus-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 90, column: 6)
--terra-tabs-modular-focus-border-radius - packages/terra-tabs/lib/ModularTabs.module.scss (line: 91, column: 6)
--terra-tabs-modular-hover-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 77, column: 8)
--terra-tabs-modular-hover-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 78, column: 8)
--terra-tabs-modular-hover-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 79, column: 8)
--terra-tabs-modular-selection-bar-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 121, column: 6)
--terra-tabs-modular-selection-bar-height - packages/terra-tabs/lib/ModularTabs.module.scss (line: 123, column: 6)
--terra-tabs-modular-selection-bar-transition - packages/terra-tabs/lib/ModularTabs.module.scss (line: 126, column: 6)
--terra-tabs-modular-tab-bar-border-bottom-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 47, column: 6)
--terra-tabs-modular-tab-bar-border-bottom-width - packages/terra-tabs/lib/ModularTabs.module.scss (line: 49, column: 6)
--terra-tabs-modular-tab-bar-padding - packages/terra-tabs/lib/ModularTabs.module.scss (line: 50, column: 6)
--terra-tabs-structural-border-radius - packages/terra-tabs/lib/StructuralTabs.module.scss (line: 101, column: 6)

**orion-fusion-theme**

--terra-tabs-active-focus-ltr-box-shadow - packages/terra-tabs/lib/ModularTabs.module.scss (line: 116, column: 8)
--terra-tabs-active-focus-rtl-box-shadow - packages/terra-tabs/lib/ModularTabs.module.scss (line: 112, column: 8)
--terra-tabs-collapsed-modular-tab-bar-border-bottom-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 26, column: 6)
--terra-tabs-collapsed-modular-tab-bar-border-bottom-width - packages/terra-tabs/lib/ModularTabs.module.scss (line: 28, column: 6)
--terra-tabs-modular-active-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 72, column: 8)
--terra-tabs-modular-active-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 73, column: 8)
--terra-tabs-modular-active-focus-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 105, column: 6)
--terra-tabs-modular-active-focus-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 106, column: 6)
--terra-tabs-modular-active-focus-border-radius - packages/terra-tabs/lib/ModularTabs.module.scss (line: 107, column: 6)
--terra-tabs-modular-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 23, column: 4)
--terra-tabs-modular-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 65, column: 6)
--terra-tabs-modular-background-position - packages/terra-tabs/lib/ModularTabs.module.scss (line: 66, column: 6)
--terra-tabs-modular-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 68, column: 6)
--terra-tabs-modular-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 69, column: 6)
--terra-tabs-modular-disabled-hover-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 85, column: 8)
--terra-tabs-modular-extended-flex - packages/terra-tabs/lib/ModularTabs.module.scss (line: 59, column: 8)
--terra-tabs-modular-focus-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 90, column: 6)
--terra-tabs-modular-focus-border-radius - packages/terra-tabs/lib/ModularTabs.module.scss (line: 91, column: 6)
--terra-tabs-modular-hover-background-image - packages/terra-tabs/lib/ModularTabs.module.scss (line: 77, column: 8)
--terra-tabs-modular-hover-background-size - packages/terra-tabs/lib/ModularTabs.module.scss (line: 78, column: 8)
--terra-tabs-modular-hover-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 79, column: 8)
--terra-tabs-modular-selection-bar-background-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 121, column: 6)
--terra-tabs-modular-selection-bar-height - packages/terra-tabs/lib/ModularTabs.module.scss (line: 123, column: 6)
--terra-tabs-modular-selection-bar-transition - packages/terra-tabs/lib/ModularTabs.module.scss (line: 126, column: 6)
--terra-tabs-modular-tab-bar-border-bottom-color - packages/terra-tabs/lib/ModularTabs.module.scss (line: 47, column: 6)
--terra-tabs-modular-tab-bar-border-bottom-width - packages/terra-tabs/lib/ModularTabs.module.scss (line: 49, column: 6)
--terra-tabs-modular-tab-bar-padding - packages/terra-tabs/lib/ModularTabs.module.scss (line: 50, column: 6)
--terra-tabs-structural-active-z-index - packages/terra-tabs/lib/StructuralTabs.module.scss (line: 120, column: 8)
--terra-tabs-structural-box-shadow - packages/terra-tabs/lib/StructuralTabs.module.scss (line: 104, column: 6)
